<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-04T13:39:58 IST" name="Default suite" finished-at="2022-10-04T13:40:12 IST" duration-ms="14315">
    <groups>
    </groups>
    <test started-at="2022-10-04T13:39:58 IST" name="Default test" finished-at="2022-10-04T13:40:12 IST" duration-ms="14315">
      <class name="CartAmount_ValidationWith_TotalAmount_TC3">
        <test-method is-config="true" signature="Ecomm_app_base_Config()[pri:0, instance:CartAmount_ValidationWith_TotalAmount_TC3@2b40ff9c]" started-at="2022-10-04T13:39:58 IST" name="Ecomm_app_base_Config" finished-at="2022-10-04T13:40:06 IST" duration-ms="7951" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Ecomm_app_base_Config -->
        <test-method is-config="true" signature="Starting_page_url()[pri:0, instance:CartAmount_ValidationWith_TotalAmount_TC3@2b40ff9c]" started-at="2022-10-04T13:40:06 IST" name="Starting_page_url" finished-at="2022-10-04T13:40:07 IST" duration-ms="1197" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Starting_page_url -->
        <test-method signature="scroller()[pri:0, instance:CartAmount_ValidationWith_TotalAmount_TC3@2b40ff9c]" started-at="2022-10-04T13:40:07 IST" name="scroller" finished-at="2022-10-04T13:40:12 IST" duration-ms="4731" status="FAIL">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index 0 out of bounds for length 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at CartAmount_ValidationWith_TotalAmount_TC3.scroller(CartAmount_ValidationWith_TotalAmount_TC3.java:66)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- scroller -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:CartAmount_ValidationWith_TotalAmount_TC3@2b40ff9c]" started-at="2022-10-04T13:40:12 IST" name="tearDown" finished-at="2022-10-04T13:40:12 IST" duration-ms="406" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- CartAmount_ValidationWith_TotalAmount_TC3 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
